src/components/community/link-community.js

```diff
@@ -0,0 +1,31 @@
+import '@brightspace-ui/core/components/link/link.js';
+import '@brightspace-ui/core/components/button/button-subtle.js';
+import { html, LitElement } from 'lit';
+import { communityUrlFactory } from './directive/community-article.js';
+import { LanguageListenerController } from '../../controllers/language-listener/language-listener.js';
+
+class CommunityLink extends LitElement {
+
+	static get properties() {
+		return {
+			text: { type: String },
+			articleMap: { attribute: 'article-map', type: Object }
+		};
+	}
+
+	constructor() {
+		super();
+		this.langController = new LanguageListenerController(this);
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		this.communityArticleDirective = communityUrlFactory(this.articleMap);
+	}
+
+	render() {
+		return html`<d2l-link href="${this.communityArticleDirective(this.langController.language)}" target="_blank">${this.text}</d2l-link>`;
+	}
+}
+
+customElements.define('d2l-labs-link-community', CommunityLink);
```

Comment:
You're going to hate me, but now that these are both "community" things in a "community" directory, it's making me want to flip-flop and have them to be `d2l-labs-communit-link` and `d2l-labs-community-button`.

The files would then become `community-button.js` and `community-link.js`.

Reply:
Sounds good and I get it haha it makes sense